### PR TITLE
Update compilemodel.py, fixes loss=inf when labels repeat

### PR DESCRIPTION
**Pull request by usernaamee**
Bug fix for loss = inf in ctc_loss when labels are repeated. 
See: https://stackoverflow.com/questions/52430214/avoid-infinity-for-ctc-loss-in-tensorflow
Instead of using the default keras ctc_batch_cost, a custom version of the function is included.
This function uses the preprocess_collapse_repeated argument, which is set to true from our ctc_loss function.

### CODE REVIEW

**Comment by usernaamee** [comment]
This patch causes removal of all repititions while training, which is not intended.
I will submit another fix using tensorflow.
